### PR TITLE
chore(deps): bump bulk-import plugins

### PR DESCRIPTION
**Pull request by rm3l**
## Description

Bump the versions of the bulk import plugins, to fix some of the issues reported so far. And also to unblock the QE test automation.

## Which issue(s) does this PR fix

- https://issues.redhat.com/browse/RHIDP-3864
- https://issues.redhat.com/browse/RHIDP-3865
- https://issues.redhat.com/browse/RHIDP-3901
- https://issues.redhat.com/browse/RHIDP-3884
- Security fixes in the backend plugin (https://github.com/janus-idp/backstage-plugins/pull/2162) - cc @kim-tsao 

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

## How to test changes / Special notes to the reviewer

/cc @debsmita1 @subhashkhileri @invincibleJai @ciiay

### CODE REVIEW

**Comment by rm3l** [comment]
/cherry-pick 1.3.x

Just so we don´t forget.

File: dynamic-plugins/imports/package.json [hunk 1]
```diff
@@ -27,8 +27,8 @@
     "@janus-idp/backstage-plugin-quay": "1.11.3",
     "@janus-idp/backstage-plugin-rbac": "1.29.4",
     "@janus-idp/backstage-plugin-tekton": "3.12.3",
-    "@janus-idp/backstage-plugin-bulk-import": "1.4.6",
-    "@janus-idp/backstage-plugin-bulk-import-backend": "1.4.1",
+    "@janus-idp/backstage-plugin-bulk-import": "1.4.7",
+    "@janus-idp/backstage-plugin-bulk-import-backend": "1.4.3",
```
Comment (rm3l) [inline dynamic-plugins/imports/package.json:31]:
```suggestion
    "@janus-idp/backstage-plugin-bulk-import-backend": "1.5.0",
```

New version of the bulk import backend plugin available; it adds server-side search https://issues.redhat.com/browse/RHIDP-3884 (https://github.com/janus-idp/backstage-plugins/pull/2140)

**Comment by rm3l** [comment]
E2E tests should be fixed with https://github.com/janus-idp/backstage-showcase/pull/1576 according to @subhashkhileri .

**Review by nickboldt — APPROVED** [review]
LGTM but tests are failing

**Comment by rm3l** [comment]
> LGTM but tests are failing

Yeah, just discussed this with @subhashkhileri.
[`ae9e6c2` (#1605)](https://github.com/janus-idp/backstage-showcase/pull/1605/commits/ae9e6c2e9d65ad6fd4305da5deb6c581e4c0d9b8) should fix the tests.

**Pull request merged**